cypress/integration/org_events_page.spec.ts

```diff
@@ -5,10 +5,14 @@ describe('/o/[orgId]/events', () => {
         cy.contains('My Organization');
     });
 
-    xit('contains events', () => {
+    xit('contains events which are linked to event pages', () => {
         // TODO: Figure out why this fails on GitHub
         cy.visit('/o/1/events');
         cy.get('[data-test="event"]').should('be.visible');
+        cy.get('[data-test="more-info-button-22"]')
```

Comment:
Yes, I thought about using that, but since all buttons have the same string from `FormattedMessage` I wonder how we get a hold of a specific button by using `findByText`?